packages/lodestar/src/executionEngine/http.ts

```diff
@@ -26,7 +26,8 @@ import {
 export type ExecutionEngineHttpOpts = {
   urls: string[];
   timeout?: number;
-  jwtSecretHex?: string;
+  /** 256 bit jwt secret in hex format without the leading 0x */
```

Comment:
If it's a hex string having hex in the variable name is really helpful. Also please prefix with 0x, everywhere in Lodestar all hex strings are '0x' prefixed. There's no good reason to not do this here

Reply:
Also, if you parse the jwt in the cli as Uint8Array and pass that of parameter you remove all this need for conversions back and forth

Reply:
actually I did, but there was an issue as it got converted to a object till it reached the execution engine

Reply:
`deepmerge` of the beacon node options in `beaconNodeOptions.getWithDefaults` causes it to become an object from for 1.g: `[0,1,2,3,4,..]` to `{"0":0,"1":1,"2":2..}`

Reply:
Oh right, deepmerge